File: frontend/src/types/attempt.ts
export type QuizAttempt = {
  id: string;
  classId: string;
  quizId: string;
  lectureTitle: string;
  questionIds: string[];
  answers: number[];
  correctIndices: number[];
  score: number;
  total: number;
  completedAt: number;
};

File: backend/internal/domain/attempt.go
package domain

type Attempt struct {
	ID             string
	ClassID        string
	QuizID         string
	LectureTitle   string
	QuestionIDs    []string
	Answers        []int
	CorrectIndices []int
	Score          int
	Total          int
	CompletedAt    int64
}
